GetnadTakeConsoleApp/Program.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Net.Http.Headers;
+
+static void Main()
+{
+    RunAsync().GetAwaiter().GetResult();
+}
+
+static async Task RunAsync()
+{
+
+    HttpClient client = new()
+    {
+        BaseAddress = new Uri("https://localhost:7167/api")
+    };
+    client.DefaultRequestHeaders.Accept.Clear();
+    client.DefaultRequestHeaders.Accept.Add(
+        new MediaTypeWithQualityHeaderValue("application/json"));
+
+    try
+    {
+        var products = await client.GetAsync(client.BaseAddress + "/products");
```

Comment:
Just an advice:  usage of string interpolation is more preferable for C#, so, your code will look like
```suggestion
        var products = await client.GetAsync($"{client.BaseAddress}/products");
```